e2e-tests/insurance/cypress/e2e/journeys/declarations/how-your-data-will-be-used/save-and-back.spec.js

```diff
@@ -66,8 +63,8 @@ context('Insurance - Declarations - How your data will be used page - Save and g
       cy.assertUrl(expected);
     });
 
-    it(`should retain the status of task 'declarations' as '${IN_PROGRESS}'`, () => {
-      cy.checkTaskStatus(task, IN_PROGRESS);
+    it('should retain the status of task `declarations` as `in progress`', () => {
```

Comment:
should the name of the task status still not come from the constant?

Reply:
For task checking I don't think it should, they're very unlikely to change and if we use constants for the task names in test descriptions, each test with task assertions has to import and destructure them

What do you think?